tests/pdctl/cluster/cluster_test.go

```diff
@@ -63,14 +66,24 @@ func (s *clusterTestSuite) TestClusterAndPing(c *C) {
 	echo := pdctl.GetEcho([]string{"-u", pdAddr, "--cacert=ca.pem", "cluster"})
 	c.Assert(strings.Contains(echo, "no such file or directory"), IsTrue)
 
-	// cluster status
-	args = []string{"-u", pdAddr, "cluster", "status"}
+	// cluster info
+	args = []string{"-u", pdAddr, "cluster"}
```

Comment:
replaced the original command?

Reply:
No, the test here used a wrong command before. 🤔